desktop/test/e2e/specs/login.js

```diff
@@ -95,26 +95,28 @@ describe( 'User Can log in', () => {
 		await mainWindow.keyboard.press( 'Enter' );
 		await mainWindow.fill( '#password', process.env.E2EPASSWORD );
 
+		await mainWindow.click( 'button:has-text("Log In")' );
+		await mainWindow.waitForSelector( '.is-logged-in' );
 		// Wait for response from the Login endpoint.
-		const [ response ] = await Promise.all( [
-			mainWindow.waitForResponse( '**/wp-login.php?action=login-endpoint' ),
-			mainWindow.click( 'button:has-text("Log In")' ),
-		] );
+		// const [ response ] = await Promise.all( [
+		// 	mainWindow.waitForResponse( '**/wp-login.php?action=login-endpoint' ),
```

Comment:
Are these supposed to be part of the change or maybe you forgot to uncomment after testing something?

Reply:
Ah, I see you added a commit `Attempt different logged-in testing approach`, so it seems to be intentional.